### ISSUE TITLE
Use `docopt` for a better CLI

### ISSUE THREAD

**Issue by kishorbhat** [therealkbhat/mazerunner#11]
We can leverage [docopt](http://docopt.org/) to make `generator` and `mazerunner` usage more intuitive and easy to use. For example:

``` bash
$ python mazerunner/generator.py -o <filename> -w <width> -h <height>
```

and

``` bash
$ python mazerunner/mazerunner.py -m <filename>
```

would be nice to see.

**Comment by giritheja** [comment]
#12 should solve this issue.

**Comment by kishorbhat** [comment]
Resolved in #12.
